### ISSUE TITLE
ofe install fails on Windows 8.1

### ISSUE THREAD

**Issue by diegosantosrd** [trevnorris/node-ofe#14]
When I try to install ofe, this message is shown:

```
LINK : fatal error LNK1194: cannot delay-load 'node.exe' due to import of data symbol '"__declspec(dllimport) const v8::OutputStream::`vftable'" (__imp_??_7OutputStream@v8@@6B@)'; link without /DELAYLOAD:node.exe [D:\and-dev\testtool-server\testtoolserver\server\node_modules\ofe\build\ofe.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Users\Diego\AppData\Roaming\npm\node_modules\npm\node_modules\node-gyp\lib\build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:204:12)
gyp ERR! System Windows_NT 6.3.9600
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Diego\\AppData\\Roaming\\npm\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd D:\and-dev\testtool-server\testtoolserver\server\node_modules\ofe
gyp ERR! node -v v6.2.0
gyp ERR! node-gyp -v v3.3.1
gyp ERR! not ok
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Diego\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install" "ofe"
npm ERR! node v6.2.0
npm ERR! npm  v2.15.6
npm ERR! code ELIFECYCLE
```

**Comment by trevnorris** [comment]
Sorry, I don't have a machine to reproduce this. Will leave this open and hopefully someone else can take a look.

**Comment by stiv-yakovenko** [comment]
someone please help :) I have virtual box if it may help

**Comment by trevnorris** [comment]
@stiv-yakovenko Just published v0.5.1. Let me know if that fixed it for you.

**Comment by trevnorris** [comment]
Will take another look if someone else reports this issue.
